pybaseball/statcast_fielding.py

```diff
@@ -36,6 +36,30 @@ def statcast_outs_above_average(year: int, pos: Union[int, str], min_att: Union[
 	data = sanitize_statcast_columns(data)
 	return data
 
+@cache.df_cache()
+def statcast_fielding_run_value(year: int, pos: Union[int, str], min_inn: int = 100) -> pd.DataFrame:
```

Comment:
Maybe we could also add `team` in here? Since Savant also provide team selection.
ref: https://baseballsavant.mlb.com/leaderboard/fielding-run-value?year=2023&team=108&min=0.1&pos=

The default value could be `team: Optional[str] = None`

Reply:
I see! Thank you for letting me know! I just realize that we can still filter team data without the argument since it also returns `team_id`